lib/src/typedefs.dart

```diff
@@ -29,6 +29,10 @@ typedef DateWidgetBuilder = Widget Function(
   DateTime date,
 );
 
+typedef WeekNumberBuilder = Widget? Function(
+  int weekNumber,
```

Comment:
Can you provide DateTime instead of weekNumber. So user can provide custom logic.